web/channels/room_channel.ex

```diff
@@ -12,8 +12,21 @@ defmodule Swolen.RoomChannel do
   end
 
   def handle_in("new_msg", %{"body" => "/don " <> item}, socket) do
-    message = "#{socket.assigns.username} has donned a(n) #{item}"
+    username = socket.assigns.username
+    message = "#{username} has donned a(n) #{item}"
+    Swolen.UserState.set(username, item)
     broadcast!(socket, "new_msg", %{body: message, from: "🌎"})
+    {:noreply, socket}
```

Comment:
This was causing an error, thought not exactly part of the functionality in this pr